types/accessRequestSearchRequest.js

```diff
@@ -0,0 +1,53 @@
+const Serializable = require('./serializable')
+
+class AccessRequestSearchRequest extends Serializable {
+  constructor(filterByRequestorIDs, filterByGroupIDs, nextToken, limit) {
+    super()
+    this.filterByRequestorIDs = filterByRequestorIDs
+    this.filterByGroupIDs = filterByGroupIDs
+    this.nextToken = nextToken
+    this.limit = limit
+  }
+
+  serializable() {
+    let toSerialize = {
+      access_request_search_filters: {
+        access_controlled_group_ids: this.filterByGroupIDs,
+        requestor_ids: this.filterByRequestorIDs,
+      },
+      next_token: this.nextToken,
+      limit: this.limit,
+    }
+
+    const serializedKeys = Object.keys(toSerialize)
+    for (const key of serializedKeys) {
+      if (toSerialize[key] === null) {
+        delete toSerialize[key]
+      }
+    }
```

Comment:
idk if you need to act on it now.. but just a note that `removeNulls` is a great candidate for an extracted utility function